theme/style.css

```diff
@@ -102,13 +88,16 @@ img {
   color: #fff;
 }
 
-.content > div:nth-child(6) {
+
+/* affecting michael's code */
+
+/*.content > div:nth-child(6) {
```

Comment:
this code is affecting how you've placed your block because you and Jasmine have swapped spots back and forth a few times. These styles are specific to Jasmine, so let's put your block of HTML after hers, let you clobber both of your styles.